doc/release/release_dev.rst

```diff
@@ -54,16 +147,23 @@ API Changes
   addition to the API change this may cause a performance regression for
   large graphs.
 - [`#4384 <https://github.com/networkx/networkx/pull/4384>`_]
-  Added edge_key parameter for MultiGraphs in to_pandas_edgelist
+  Added ``edge_key`` parameter for MultiGraphs in to_pandas_edgelist
+- [`#4461 <https://github.com/networkx/networkx/pull/4461>`_]
+  Added ``create_using`` parameter to ``binomial_tree``
 - [`#4466 <https://github.com/networkx/networkx/pull/4466>`_]
   `relabel_nodes` used to raise a KeyError for a key in `mapping` that is not
   a node in the graph, but it only did this when `copy` was `False`. Now
   any keys in `mapping` which are not in the graph are ignored.
+- [`#4502 <https://github.com/networkx/networkx/pull/4502>`_]
+  Moves ``maximum_independent_set`` to the ``clique`` module in ``approximation``.
+- [`#4536 <https://github.com/networkx/networkx/pull/4545>`_]
```

Comment:
This is in the deprecations.  Does it need to be here.  also, 4536 at the end not 4545

Reply:
I had decided to put it in both sections since it involves both a deprecation and a new function. The bullet in `API changes` includes an extra sentence explaining why the original functions are being replaced.

I don't have a strong opinion though - feel free to go ahead and remove it if you prefer!